pages/index.tsx

```diff
@@ -35,10 +38,8 @@ export default function Home() {
       setPosts((prev) => [...prev, ...posts]);
       if (posts.length > 10) {
         setHasMore(true);
-      } else if (posts.length === 0) {
-        setHasMore(true);
       } else {
-        setHasMore(false);
+
       }
```

Comment:
Shouldn't you have removed the else block

Reply:
Will check it...